meson.build

```diff
@@ -295,7 +296,7 @@ subdir('tests')
 #
 version_file = vcs_tag(input : 'src/version.cpp.in', output : 'version.cpp')
 executable('dosbox', ['src/main.cpp', 'src/dosbox.cpp', version_file],
-           dependencies : [threads_dep, sdl2_dep] + internal_deps,
+           dependencies : [atomic_dep, threads_dep, sdl2_dep] + internal_deps,
```

Comment:
Hmm, for a moment here I thought that this dep will end up as `misc` dependency, but you're right here - `atomic` is more like `threads` - it's distributed with the compiler, so placing it top-level makes a bit more sense.

Reply:
Yes; I did try one CI round with it as a `misc` dependency, but linking of `dosbox` failed (https://user-images.githubusercontent.com/1557255/110025514-537caf00-7ce4-11eb-9561-836ee36ad01e.png), so had to move the dependency to the executable, which finally worked across all the CI jobs.